cmd/apply/status/status_test.go

```diff
@@ -46,7 +46,8 @@ func setupKustomize(t *testing.T) string {
 	err = ioutil.WriteFile(filepath.Join(f, "kustomization.yaml"), []byte(`apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 configMapGenerator:
-- name: testMap
+- name: testmap
```

Comment:
nit: `test-map`
I think camel case is not commonly used for YAML manifest. Correct me if I am wrong.